app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Checkbox, Button, Avatar, Tooltip } from "neetoui";
+
+export default function ContactTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  setShowDeleteAlert,
+  contacts = [],
+}) {
+  const handleContactDelete = contactId => {
+    setShowDeleteAlert(true);
+    setSelectedContactIds([contactId]);
+  };
+
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox">
```

Comment:
Please add classnames `nui-table--hover nui-table--actions` to the table.
This adds hover effects for table row and hover effects for `IconButtons`.